### PR TITLE
Create 2024-03.md

### LINKED ISSUE #37: Issue 13 of the newsletter
This is a call to everyone out there who works in open bioinformatics - please share interesting stuff! This could be a new project launch, an interesting update or release, a link to useful resources (e.g. some nice biology related icons or an open source tool that could be useful, etc. If you aren't sure about something, feel free to drop me a line personally to discuss it, e.g. via Mastodon (@gedankenstuecke@scholar.social or email (bgreshake@gmail.com).

Promoting yourself or your workplace is totally okay here - this is literally what the newsletter is for, within reason!

The aim is to send out this newsletter around December/January

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Starting the draft of the next newsletter (closes #37 when merged). 

There's a few bits and bobs where I could help with:

- The ISCBacademy events: @nlharris would you be the right person to ask for links to the recordings on YouTube?
- The CWL conference in the Netherlands: @mr-c you added it into the last newsletter as 'save the date', is there a registration or any updates worth putting into this one?

Tentatively, this should go out in early March (but I'm not sure prior to March 5, the date of the ISCBacademy talk is realistic?)

### CODE REVIEW

**Comment by yochannah** [comment]
ooh - would y'all be interested in a shout-out to cohort-based open science 101 NASA training? applications currently open :) https://openlifesci.org/nasa/

**Comment by gedankenstuecke** [comment]
@yochannah sure, that sounds great! What's the application deadline for those trainings? As I think most realistically the newsletter won't go out before early March (I hope it doesn't take that long, but just to tone down expectations 😂 )

**Comment by mr-c** [comment]
> * The CWL conference in the Netherlands: @mr-c you added it into the last newsletter as 'save the date', is there a registration or any updates worth putting into this one?

Updated website: https://www.commonwl.org/conferences/2024/

The 2024 Common Workflow Language Conference will be held primarily on the campus of VU Amsterdam, in the Netherlands, on Wednesday & Thursday May 15th & 16th. Following is a contribution-a-thon on that Friday. The conference talks will be also presented online, and there will be two online-only sessions the week prior at times convenient elsewhere in the world.

The call for submissions will open later in February 2024 with signups for attendance to follow.

**Comment by gedankenstuecke** [comment]
Thanks, added it in @mr-c!

**Comment by yochannah** [comment]
😁 Thanks Bastian! @iramosp, could you work out a nice cohort advertising blurb? I guess it'll be for Cohort 2 onwards...

**Comment by iramosp** [comment]
Here is our contribution:

### Join the Nebula Open Science Program by OLS and NASA
The [Nebula program](https://openlifesci.org/nasa/) is a dynamic 6-week online training designed for individuals and teams eager to integrate open science practices into their research. In collaboration with [NASA TOPS](https://nasa.github.io/Transform-to-Open-Science/), OLS will offer five cohorts per year. Benefit from expert guidance, hands-on learning and tailored support to advance your projects. The next round of applications opens in late March.

**Comment by gedankenstuecke** [comment]
Thanks, added it in!

File: newsletters/2024-03.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2024-03.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter March 2024
+
+TL;DR: In this newsletter we provide key dates for BOSC 2024; share the upcoming OBF event fellowship, highlight an upcoming ISCBacademy event, and share upcoming community opportunities.
```
Comment (nlharris) [inline newsletters/2024-03.md:11]:
```suggestion
TL;DR: In this newsletter we provide key dates for BOSC 2024, share the upcoming OBF event fellowship application deadline, highlight an upcoming ISCBacademy talk, and share upcoming community opportunities.
```

File: newsletters/2024-03.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+In longer form: 
+
+## OBF News
+
+### BOSC news
+
+The 25th(!) annual Bioinformatics Open Source Conference (BOSC 2024) will take place on July 15/16 as part of ISMB 2024 in Montréal, Canada, and online. As in prior years, there will be a _CollaborationFest_, a collaborative work event (including but not limited to hacking) happening around BOSC, this year right after the event on July 17/18.
+
+There is still time to submit your abstracts for posters and talks. And there [is even an early poster submission/acceptance route that you can use until March 21](https://www.open-bio.org/2024/02/18/early-poster-acceptance-2024/). With this we want to make it easier for people to support them in their travel planning.
```
Comment (nlharris) [inline newsletters/2024-03.md:21]:
```suggestion
There is still time to submit your abstracts for posters and talks. And there [is even an early poster submission/acceptance route that you can use until March 21](https://www.open-bio.org/2024/02/18/early-poster-acceptance-2024/). With this we want to make it easier for people to plan their travel to BOSC/ISMB.
```

File: newsletters/2024-03.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The actual conference dates are:
+
+* July 12-16, 2024: ISMB 2024 (Montréal, Canada and online)
+* July 15-16, 2024: BOSC 2024 (part of ISMB)
+* July 17-18: Post-BOSC CollaborationFest
+
+
+### OBF Event Fellowships
+
+Next round: April 1, 2024
```
Comment (nlharris) [inline newsletters/2024-03.md:40]:
```suggestion
Next deadline: April 1, 2024
```

File: newsletters/2024-03.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+The current application period for the OBF Event Fellowship lasts until April 1, 2024. So there is still some [time to get your application submitted](https://forms.gle/3yLg94G1C9nkZkvq5).
+
+These fellowships are ideal to support your virtual or in-person BOSC attendance, but can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```
Comment (nlharris) [inline newsletters/2024-03.md:44]:
```suggestion
These fellowships are ideal to support your virtual or in-person BOSC attendance, but can also be used to cover registration fees for other relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```

File: newsletters/2024-03.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+### ISCBacademy events
+
+The OBF is about to host another _ISCBacademy webinar_: On March 5, 2024, we will host Gemma Turon who is the co-founder and CEO of the _Ersilia Open Source Initiative_. In her talk, Gemma will present on _Ersilia, open source AI/ML for (antimicrobial) drug discovery_. The [details for how to attend are on the OBF website](https://www.open-bio.org/2024/02/08/iscbacademy-gemma-turon/)
+
+Additionally, on October 3, 2023 we collaborated with the _Bio-Ontologies_ COSI and had Sierra Moxon speak about _"LinkML: an open data modeling framework, grounded with ontologies"_. The [recording of this talk is now on YouTube](https://www.youtube.com/watch?v=nyNp09WYLzw).
```
Comment (nlharris) [inline newsletters/2024-03.md:50]:
```suggestion
Our previous ISCBacademy talk was on October 3, 2023 . We collaborated with the _Bio-Ontologies_ COSI and had Sierra Moxon speak about _"LinkML: an open data modeling framework, grounded with ontologies"_. The [recording of this talk is now on YouTube](https://www.youtube.com/watch?v=nyNp09WYLzw). 
```

File: newsletters/2024-03.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+* May 13: Talk/poster acceptance notification
+* May 20: Late poster submission deadline
+
+The actual conference dates are:
+
+* July 12-16, 2024: ISMB 2024 (Montréal, Canada and online)
+* July 15-16, 2024: BOSC 2024 (part of ISMB)
+* July 17-18: Post-BOSC CollaborationFest
+
+
```
Comment (nlharris) [inline newsletters/2024-03.md:37]: To keep up with BOSC news, please follow us on [LinkedIn](https://www.linkedin.com/groups/14344023/), [Bluesky](https://bsky.app/profile/bosc.bsky.social), [Mastodon](https://genomic.social/@BOSC), and [Slack](https://join.slack.com/t/obf-bosc/shared_invite/zt-n5ur1gsj-z2C~69_4lYTFPg5tbWA8Ew)! (We are no longer on Twitter/X; [find out why](https://www.open-bio.org/2023/11/20/leaving-x/).)

Comment (nlharris) [reply newsletters/2024-03.md:37]: (I meant to add that as a new line; not sure if that worked)

**Review by nlharris — CHANGES_REQUESTED** [review]
Thank you for putting this together, @gedankenstuecke! 🙌

File: newsletters/2024-03.md (hunk 6 above)
Comment (gedankenstuecke) [reply newsletters/2024-03.md:38]: no worries, i added it manually!

**Review by nlharris — APPROVED** [review]
looks good to me!

**Comment by peterjc** [comment]
Thanks all!

**Comment by gedankenstuecke** [comment]
Oh sorry @JulieDeMan, I fear the newsletter went out yesterday evening (UTC)! The next newsletter will probably go out around the timing of BOSC (#40), if you have any things that would be relevant in June/July we'd be happy to add those!
